core/kernel/classes/class.Property.php

```diff
@@ -222,6 +226,31 @@ public function setRange(core_kernel_classes_Class $class)
         return (bool) $returnValue;
     }
 
+    public function getDependsOnProperty(): ?core_kernel_classes_Property
```

Comment:
As discussed before it would be better to operate with collections ( arrays ) but not a single elements to prevent breaking changes in a future